src/jupyter_contrib_nbextensions/nbconvert_support/__init__.py

```diff
@@ -2,7 +2,8 @@
 
 import os
 
-from .latex_envs import LenvsHTMLExporter, LenvsLatexExporter
+from latex_envs.latex_envs import LenvsHTMLExporter, LenvsLatexExporter
```

Comment:
Not sure that we have to keep these two imports since the entry points are defined in the `latex_envs.py` module.